pandas_schema/version.py

```diff
@@ -1 +1 @@
-__version__ = '0.3.5'
+__version__ = '0.3.5.7'
```

Comment:
Python doesn't support 4 levels of versioning (as far as I know). Feel free to leave this file out of the PR, and I will increment it myself before a release.